mlflow/ml-package-versions.yml

```diff
@@ -310,7 +310,9 @@ spark:
       # -Pconnect -Phive is requried by spark connect
       ./build/mvn -Pconnect -Phive -DskipTests --no-transfer-progress clean package
```

Comment:
according to discussion, we should use `sbt` instead of mvn ?

Reply:
Not in this PR.